ep0099/ep0099.go

```diff
@@ -32,7 +32,7 @@ func New(bus i2c.Bus, address uint16) (*Dev, error) {
 
 	d := &Dev{
 		i2c:   i2c.Dev{Bus: bus, Addr: address},
-		state: buildChannelsList(),
+		state: [4]State{StateOff, StateOff, StateOff, StateOff},
```

Comment:
It's the default so you can skip the whole line.

Reply:
this makes sense. will remove it.